clients/cli/cmd/internal/push.go

```diff
@@ -190,8 +194,7 @@ func (source *Source) Push(client *phrase.APIClient, waitForResults bool, branch
 				print.Failure("There was an error processing %s. Your changes were not saved online.", localeFile.RelPath())
 			}
 		} else {
-			fmt.Println("done!")
-			fmt.Printf("Check upload Id: %s, filename: %s for information about processing results.\n", upload.Id, upload.Filename)
+			print.NonBatchPrintf("done!\nCheck upload Id: %s, filename: %s for information about processing results.\n", upload.Id, upload.Filename)
```

Comment:
I think we should have a json response here as well.
How about printing the whole `upload` object?